ietf/draft-ietf-calext-jscalendar-icalendar-09.txt

```diff
@@ -0,0 +1,6272 @@
+   +---------+-----------------------------+-------------------+---------+
+   |IN-      |[RFC5545], Section 3.2.12    |accepted           |in-      |
+   |PROCESS  |                             |                   |process  |
+   +---------+-----------------------------+-------------------+---------+
+   |FAILED   |[I-D.ietf-calext-ical-tasks],|accepted           |failed   |
+   |         |Section 11.1                 |                   |         |
+   +---------+-----------------------------+-------------------+---------+
+
+      Table 18: Converting the PARTSTAT parameter in a VTODO component
+                         to Participant properties
+
+   The following examples illustrate how to convert the ATTENDEE
+   property:
+
+
+
+Stepanek                   Expires 16 May 2025                 [Page 42]
+
+Internet-Draft                 JSCalendar                  November 2024
+
+
+   ATTENDEE;PARTSTAT=TENTATIVE;CN=Henry Cabot:mailto:hcabot@example.com
+   "participants": {
+     "p3": {
+       "@type": "Participant",
+       "calendarAddress": "mailto:hcabot@example.com",
+       "name": "Henry Cabot",
+       "participationStatus": "tentative",
+       "roles": {
+         "attendee": true
+       },
+       "sendTo": {
+         "imip": "mailto:hcabot@example.com"
+       }
+     },
+     "o4": {
+       "...": ""
+     }
+   }
+
+                Figure 21: Converting the ATTENDEE property
+
+   BEGIN:PARTICIPANT
+   CALENDAR-ADDRESS:mailto:foo@example.com
+   DTSTAMP:20230723T125201Z
+   ...
+   END:PARTICIPANT
+   ATTENDEE;PARTSTAT=TENTATIVE:mailto:foo@example.com
+   "participants": {
+     "23": {
+       "@type": "Participant",
+       "calendarAddress": "mailto:foo@example.com",
+       "participationStatus": "tentative",
+       "scheduleUpdated": "2023-07-23T12:52:01Z",
+       "...": ""
+     },
+     "o4": {
+       "...": ""
+     }
+   }
+
+   Figure 22: Converting the ATTENDEE property and PARTICIPANT component
+
+   BEGIN:VTODO
+   ATTENDEE;PARTSTAT=IN-PROCESS:mailto:foo@example.com
+   ...
+
+
+
+
+
+
+Stepanek                   Expires 16 May 2025                 [Page 43]
+
+Internet-Draft                 JSCalendar                  November 2024
+
+
+   "@type": "Task",
+   "participants": {
+     "1": {
+       "@type": "Participant",
+       "calendarAddress": "mailto:foo@example.com",
+       "participationStatus": "accepted",
+       "progress": "in-process",
+       "...": ""
+     },
+     "o": {
+       "...": ""
+     }
+   }
+
+     Figure 23: Converting the ATTENDEE property with a VTODO-specific
+                            participation status
+
+2.3.5.  CALENDAR-ADDRESS
+
+   The CALENDAR-ADDRESS property [RFC9073] (Section 6.4) in a
+   PARTICIPANT component converts to the calendarAddress property
+   [I-D.ietf-jmap-calendars] (Section 5.1.1) of the Participant object.
+
+   The following example illustrates how to convert the CALENDAR-ADDRESS
+   property:
```

Comment:
The above sections seem to say that ATTENDEE fills in the calendarAddress property. I guess this overrides it?